patroni/dcs/etcd3.py

```diff
@@ -733,7 +733,7 @@ def create_lease(self) -> None:
 
     @property
     def cluster_prefix(self) -> str:
-        return self._base_path + '/' if self.is_citus_coordinator() else self.client_path('')
+        return self._base_path + '/' if self.is_mpp_coordinator() else self.client_path('')
```

Comment:
What about adding a docstring if you are touching this code? :)

Reply:
I'd appreciate if you can help with the wording ;)

Reply:
Put a suggestion.